### ISSUE TITLE
How to use the client local by default

### ISSUE THREAD

**Issue by dtrckd** [ultrabug/mkdocs-static-i18n#99]
Hello,

How could we make the client to automatically land into the language defined in its browser local if any ?

Thanks.

**Comment by ultrabug** [comment]
Hello @dtrckd ; 

I'm not sure it's the responsibility of the plugin to be honest. I think it's a theme's feature and that's where this should be asked if you agree.

Thanks

**Comment by ultrabug** [comment]
>  it would make sense to be consistent wit what see the client...

I agree with you there of course. But there are so many themes available that adding a frontend implementation on this plugin sounds like an endless path.

> do you suggest it should be asked rather at mkdocs or mkdocs-material ?

Well we at mkdocs concentrate on the engine, not so much on the themes'  features but you could lookup or start a new Discussion on mkdocs or ask mkdocs-material if that's the theme you're using.
